boa3_test/tests/compiler_tests/test_native/test_policy.py

```diff
@@ -1,84 +1,55 @@
-from boa3_test.tests.boa_test import BoaTest  # needs to be the first import to avoid circular imports
+from neo3.core import types
 
 from boa3.internal import constants
 from boa3.internal.exception import CompilerError
-from boa3.internal.neo3.vm import VMState
-from boa3_test.tests.test_drive.testrunner.boa_test_runner import BoaTestRunner
+from boa3_test.tests import boatestcase
 
 
-class TestPolicyContract(BoaTest):
+class TestPolicyContract(boatestcase.BoaTestCase):
     default_folder: str = 'test_sc/native_test/policy'
 
-    def test_get_hash(self):
-        path, _ = self.get_deploy_file_paths('GetHash.py')
-        runner = BoaTestRunner(runner_id=self.method_name())
+    async def test_get_hash(self):
+        await self.set_up_contract('GetHash.py')
 
-        invokes = []
-        expected_results = []
+        expected = types.UInt160(constants.POLICY_SCRIPT)
+        result, _ = await self.call('main', [], return_type=types.UInt160)
+        self.assertEqual(expected, result)
 
-        invokes.append(runner.call_contract(path, 'main'))
-        expected_results.append(constants.POLICY_SCRIPT)
+    async def test_get_exec_fee_factor(self):
+        await self.set_up_contract('GetExecFeeFactor.py')
 
-        runner.execute()
-        self.assertEqual(VMState.HALT, runner.vm_state, msg=runner.error)
-
-        for x in range(len(invokes)):
-            self.assertEqual(expected_results[x], invokes[x].result)
-
-    def test_get_exec_fee_factor(self):
-        path, _ = self.get_deploy_file_paths('GetExecFeeFactor.py')
-        runner = BoaTestRunner(runner_id=self.method_name())
-
-        invoke = runner.call_contract(path, 'main')
-        runner.execute()
-        self.assertEqual(VMState.HALT, runner.vm_state, msg=runner.error)
-        self.assertIsInstance(invoke.result, int)
+        result, _ = await self.call('main', [], return_type=int)
+        self.assertIsInstance(result, int)
```

Comment:
now that we're using `boaconstructor` and `mamba`, we can get the expected result with `neo3.api.wrappers.PolicyContract` helper methods
In cases we cannot assert the actual expected result, instead of checking the type you could check the value. If the type is wrong the error will be raised on `call` method
here could be replaced by `self.assertGreater(result, 0)`